packages/@sanity/cli/src/commands/blueprints/initBlueprintsCommand.ts

```diff
@@ -0,0 +1,96 @@
+import {join} from 'node:path'
+import {cwd} from 'node:process'
+
+import {type CliCommandDefinition} from '../../types'
+
+const helpText = `
+Examples
+  # Create a new Blueprint manifest file
+  sanity blueprints init
+`
+
+const initBlueprintsCommand: CliCommandDefinition = {
+  name: 'init',
+  group: 'blueprints',
+  helpText,
+  signature: '',
+  description: 'Initialize a new Blueprint manifest file',
+  hideFromHelp: true,
+  async action(args, context) {
+    const {apiClient, output, prompt} = context
+    const {print} = output
+
+    const client = apiClient({
+      requireUser: true,
+      requireProject: false,
+    })
+    const {token} = client.config()
+
+    if (!token) {
+      print('No API token found. Please run `sanity login` first.')
+      return
+    }
+
+    const {blueprint: blueprintAction, projects: projectsAction} = await import(
+      '@sanity/runtime-cli/actions/blueprints'
+    )
+
+    const existingBlueprint = blueprintAction.findBlueprintFile()
+
+    if (existingBlueprint) {
+      print(`A blueprint file already exists: ${existingBlueprint.fileName}`)
+      return
+    }
+
+    const blueprintExtension = await prompt.single({
+      type: 'list',
+      message: 'Choose a Blueprint manifest file type:',
+      choices: [
+        {value: 'json', name: 'JSON (Recommended)'},
+        {value: 'js', name: 'JavaScript (Beta)'},
+        {value: 'ts', name: 'TypeScript (Alpha)'},
```

Comment:
Again, should we expose these options?

Reply:
Technically they're all supported. But I think removing the option for dynamic blueprints is a good idea right now. thanks